src/node.rs

```diff
@@ -229,6 +229,8 @@ impl fmt::Debug for Text {
 pub type Attributes = indexmap::IndexMap<QualName, StrTendril>;
 
 /// A Map of attributes that doesn't preserve the order of the attributes.
+/// Please enable the `deterministic` feature for order-preserving
```

Comment:
I think the newline is necessary – otherwise wouldn't this render on the same line (not 100% sure though)

```suggestion
/// 
/// Please enable the `deterministic` feature for order-preserving
```